src/core/Note.cpp

```diff
@@ -47,7 +47,8 @@ Note::Note( const TimePos & length, const TimePos & pos,
 	m_panning( qBound( PanningLeft, panning, PanningRight ) ),
 	m_length( length ),
 	m_pos( pos ),
-	m_detuning( nullptr )
+	m_detuning( nullptr ),
```

Comment:
I ended up keeping this change out, since the PR doesn't involve the detuning member. The line is only changed because I needed to add a comma, maybe since it's a member from another feature the change should be done separately, just to make sure the it doesn't affect other code?

Reply:
> I ended up keeping this change out, since the PR doesn't involve the detuning member. The line is only changed because I needed to add a comma, maybe since it's a member from another feature the change should be done separately, just to make sure the it doesn't affect other code?

Oh, I thought you had initialized ``m_detuning`` to be ``nullptr`` in the member wise initialization list. In that case, I would say just fix the formatting on that line. This doesn't need to be an in-class initializer at least for this PR, even though it can be.